apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
+    // Demo getRegistryAnswer
+    const getRegistryAnswer = (id) => {
+      // How do we get the registry answer?
+      return "true";
+    };
```

Comment:
What we'll have with autofill is a `autofill` prop on the question with value `"monument"` for example. It also has a `answer` of course. There is no way (yet) to see if the user has overridden the register value. It also depends if we want the user to allow for that.

Reply:
Thanks!